.cirrus.yml

```diff
@@ -1,5 +1,6 @@
 env:
   CC_TEST_REPORTER_ID: ENCRYPTED[!0f645db5296da5f9df5391329da33feacafcda008fc03b66a851dcea7d350994733ca486fc381159fdee966eecbd46e7!]
+  MAP_API_KEY: ENCRYPTED[!f9b96f6e8a3d7d33c6af297ea97874cadb1c7ff4f5184023a4f0e3df4c82302fdeaa2fc0eff5734955bf8011104de5b0!]
```

Comment:
Finally! Very nice.

We could regenerate a new key to be absolutely certain (because the old one is still in the git)